### ISSUE TITLE
[Feature request] Translations

### ISSUE THREAD

**Issue by jorgesumle** [arzzen/git-quick-stats#75]
It's only available in English.

**Comment by arzzen** [comment]
Which languages would you like to see on git-quick-stats?

**Comment by jorgesumle** [comment]
I would like to see Esperanto, Spanish and German. I could help with Esperanto, once the translation support is done.

**Comment by pulponair** [comment]
@arzzen would you mind creating another issue "introduce translation support" #hacktoberfest. I could take this one as well as german translation.

**Comment by arzzen** [comment]
@pulponair issue created, you can take it, thanks
